Documentation/dev-guide/interacting_v3.md

```diff
@@ -267,14 +314,29 @@ bar
 PUT
 foo
 bar_new
+```
 
+```bash
 # watch for changes on key `foo` since revision 3
 $ etcdctl watch --rev=3 foo
 PUT
 foo
 bar_new
 ```
 
+Here is an example to watch only from the last historical change:
+
+```bash
```

Comment:
@xiang90 i realized my mistake. Intead of having only key as foo i made it wrongly to foo_latest. Will correct it and also add inline comments.
Thank you!